src/BuildingRegistry.Projections.Syndication/Address/AddressPersistentLocalIdProjection.cs

```diff
@@ -50,15 +50,18 @@ public AddressPersistentLocalIdProjection()
 
         private static async Task AddSyndicationItemEntry(AtomEntry<SyndicationContent<Address>> entry, SyndicationContext context, CancellationToken ct)
         {
+            var addressId = Guid.Parse(entry.Content.Object.AddressId);
```

Comment:
should we be using TryParse instead?

Reply:
I've thought about, but when it's false it needs to crash anyway.
The events we subscribe to have a `Guid` as addressid.